FoxHound.App/Blogs/CreateBlog/CreateBlogCommandValidator.cs

```diff
@@ -0,0 +1,13 @@
+﻿using FluentValidation;
+
+namespace FoxHound.App.Blogs.CreateBlog
+{
+    public class CreateBlogCommandValidator : AbstractValidator<CreateBlogCommand>
+    {
+        public CreateBlogCommandValidator()
+        {
+            RuleFor(x => x.Owner)
+                .NotEmpty().WithMessage("Owner is required");
```

Comment:
This looks good, but consider adding a max length validation as well.

Reply:
Agree